Ingest search docs no longer working on ubuntu latest

As of 2025-02-03, ubuntu latest image is 24.04.1, and no longer seems to ship with the heroku CLI. So the CD workflow is breaking when it tries to ingest search documents, example: https://github.com/danielabar/meblog/actions/runs/13111964885/job/36577680704

```
2025-02-03T11:03:41.4246369Z ##[group]Run HEROKU_API_KEY=$HEROKU_API_KEY cat search.sql | heroku pg:psql $HEROKU_POSTGRES_ADDON_NAME --app $HEROKU_APP_NAME
2025-02-03T11:03:41.4247170Z [36;1mHEROKU_API_KEY=$HEROKU_API_KEY cat search.sql | heroku pg:psql $HEROKU_POSTGRES_ADDON_NAME --app $HEROKU_APP_NAME[0m
2025-02-03T11:03:41.4275371Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2025-02-03T11:03:41.4275712Z env:
2025-02-03T11:03:41.4275954Z   HEROKU_APP_NAME: ***
2025-02-03T11:03:41.4276234Z   HEROKU_API_KEY: ***
2025-02-03T11:03:41.4276523Z   HEROKU_POSTGRES_ADDON_NAME: ***
2025-02-03T11:03:41.4276775Z ##[endgroup]
2025-02-03T11:03:41.4330969Z /home/runner/work/_temp/53fc411a-797d-4b9d-88b8-4f86573d3d71.sh: line 1: heroku: command not found
2025-02-03T11:03:41.4337961Z cat: write error: Broken pipe
2025-02-03T11:03:41.4350404Z ##[error]Process completed with exit code 127.
```

Might need an additional step just before this to install it on the runner if possible.

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -45,6 +45,14 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./public
 
+      # https://devcenter.heroku.com/articles/heroku-cli#install-with-ubuntu-debian-apt-get
+      - name: Install Heroku CLI
+        run: |
+          curl https://cli-assets.heroku.com/install-ubuntu.sh | sh
+
+      - name: Verify Heroku CLI Installation
+        run: heroku --version
+
       - name: Ingest search docs
         env:
           HEROKU_APP_NAME: ${{ secrets.HEROKU_APP_NAME }}

